src/serialisation.rs

```diff
@@ -28,16 +31,18 @@ pub enum SerialisationError {
 
 /// Function to serialise an Encodable type
 pub fn serialise<T>(data: &T) -> Result<Vec<u8>, SerialisationError>
-                                 where T: ::rustc_serialize::Encodable {
-    let mut encoder = ::cbor::Encoder::from_memory();
+    where T: Encodable
+{
```

Comment:
Is this done by rustfmt ? Otherwise I think we follow the practice of *not* opening function body on a different line